work7/task7_3.py

```diff
@@ -0,0 +1,6 @@
+def square(num):
+    return num ** 2
+
+numbers = [1, 2, 3, 4, 5]
+squares = list(map(square, numbers))
```

Comment:
you use default map. I want you to implement your own one

Reply:
fix